src/script/conversation/ConversationRepository.ts

```diff
@@ -153,6 +153,11 @@ type FetchPromise = {rejectFn: (error: ConversationError) => void; resolveFn: (c
 type EntityObject = {conversationEntity: Conversation; messageEntity: Message};
 type IncomingEvent = ConversationEvent | ClientConversationEvent;
 
+export enum CONVERSATION_READONLY_STATE {
+  READONLY_SELF_DOES_NOT_SUPPORT_MLS = 'READONLY_SELF_DOES_NOT_SUPPORT_MLS',
```

Comment:
How about naming it READONLY_ONE_TO_ONE_SELF_DOES_NOT_SUPPORT_MLS, to be more clear that this case of conversation being readonly can happen for 1:1 conversation only?

Reply:
I would like to merge this PR and test its functionality now. We can rename it to something better later. READONLY_ONE_TO_ONE_SELF_DOES_NOT_SUPPORT_MLS is really long, READONLY_ONE_TO_ONE_SELF_UNSUPPORTED_MLS is better